zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -220,6 +221,16 @@ def get_all(kube_client, kube_func, namespace=None, **kwargs) -> list:
     return items
 
 
+def parse_resources(resources):
```

Comment:
We are not handling the ``ValueError`` exception anywhere, and that would block the discovery of further resources - regardless of whether an invalid "limit" can make it to Kubernetes.

Reply:
Good point, although I don't think it's possible to read invalid values out.